src/app/components/Metadata/__snapshots__/index.test.jsx.snap

```diff
@@ -1,1623 +1,15 @@
-    name="og:title"
-  />
-  <meta
-    content="article"
-    name="og:type"
-  />
-  <meta
-    content="https://www.bbc.com/news/articles/c0000000001o"
-    name="og:url"
-  />
-  <meta
-    content="summary_large_image"
-    name="twitter:card"
-  />
-  <meta
-    content="@BBCNews"
-    name="twitter:creator"
-  />
-  <meta
-    content="This is a description"
-    name="twitter:description"
-  />
-  <meta
-    content="BBC News"
-    name="twitter:image:alt"
-  />
-  <meta
-    content="https://www.bbc.com/news/image.png"
-    name="twitter:image:src"
-  />
-  <meta
-    content="@BBCNews"
-    name="twitter:site"
-  />
-  <meta
-    content="An article title - BBC News"
-    name="twitter:title"
-  />
-  <link
-    href="https://foo.com/static/news/image.png"
-    rel="apple-touch-icon"
-  />
-  <link
-    href="https://news.files.bbci.co.uk/include/articles/public/news/images/icons/icon-72x72.png"
-    rel="apple-touch-icon"
-    sizes="72x72"
-  />
-  <link
-    href="https://news.files.bbci.co.uk/include/articles/public/news/images/icons/icon-96x96.png"
-    rel="apple-touch-icon"
-    sizes="96x96"
-  />
-  <link
-    href="https://news.files.bbci.co.uk/include/articles/public/news/images/icons/icon-128x128.png"
-    rel="apple-touch-icon"
-    sizes="128x128"
-  />
-  <link
-    href="https://news.files.bbci.co.uk/include/articles/public/news/images/icons/icon-144x144.png"
-    rel="apple-touch-icon"
-    sizes="144x144"
-  />
-  <link
-    href="https://news.files.bbci.co.uk/include/articles/public/news/images/icons/icon-152x152.png"
-    rel="apple-touch-icon"
-    sizes="152x152"
-  />
-  <link
-    href="https://news.files.bbci.co.uk/include/articles/public/news/images/icons/icon-192x192.png"
-    rel="apple-touch-icon"
-    sizes="192x192"
-  />
-  <link
-    href="https://news.files.bbci.co.uk/include/articles/public/news/images/icons/icon-384x384.png"
-    rel="apple-touch-icon"
-    sizes="384x384"
-  />
-  <link
-    href="https://news.files.bbci.co.uk/include/articles/public/news/images/icons/icon-512x512.png"
-    rel="apple-touch-icon"
-    sizes="512x512"
-  />
-  <link
-    href="https://news.files.bbci.co.uk/include/articles/public/news/images/icons/icon-72x72.png"
-    rel="icon"
-    sizes="72x72"
-    type="image/png"
-  />
-  <link
-    href="https://news.files.bbci.co.uk/include/articles/public/news/images/icons/icon-96x96.png"
-    rel="icon"
-    sizes="96x96"
-    type="image/png"
-  />
-  <link
-    href="https://news.files.bbci.co.uk/include/articles/public/news/images/icons/icon-192x192.png"
-    rel="icon"
-    sizes="192x192"
-    type="image/png"
-  />
-  <link
-    href="https://news.files.bbci.co.uk/include/articles/public/news/images/icons/icon-512x512.png"
-    rel="apple-touch-startup-image"
-  />
-  <link
-    href="/favicon.ico"
-    rel="shortcut icon"
-    type="image/x-icon"
-  />
-</HelmetWrapper>
-`;
+exports[`Metadata News AMP article should render correctly 1`] = `<div />`;
```

Comment:
Snapshot coverage may be restored after PR #[3694](https://github.com/bbc/simorgh/pull/3694)